dev-lisp/abcl/abcl-1.9.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="Armed Bear Common Lisp is a Common Lisp implementation for the JVM"
+HOMEPAGE="https://abcl.org"
+SRC_URI="https://abcl.org/releases/${PV}/abcl-src-${PV}.tar.gz"
+
+LICENSE="GPL-2-with-classpath-exception"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=virtual/jdk-1.8:*"
+RDEPEND=">=virtual/jre-1.8:*"
+
+S="${WORKDIR}/abcl-src-${PV}"
+
+src_compile() {
+	eant abcl.compile || die "Can't compile ABCL"
```

Comment:
Looking at `eant` in java-utils-2.eclass, it looks like it dies for us (i.e. || die here is unreachable unless using nonfatal).